src/urllib3/util/request.py

```diff
@@ -123,7 +132,7 @@ def set_file_position(
     return pos
 
 
-def rewind_body(body: IO[AnyStr], body_pos: Optional[Union[int, object]]) -> None:
+def rewind_body(body: IO[AnyStr], body_pos: _TYPE_BODY_POSITION) -> None:
```

Comment:
This would remain the same, but the None test would also need the ignore clause.